src/main/kotlin/com/example/mykku/auth/tool/JwtTokenProvider.kt

```
@@ -0,0 +1,73 @@
+                email = userEmail,
+                nickname = member.nickname,
+                profileImage = member.profileImage
+            )
+        )
+    }
+
+    fun validateToken(token: String): Boolean {
+        return try {
+            parseToken(token)
+            true
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)